.github/workflows/release.yml

```
@@ -0,0 +1,77 @@
+        run: |
+          if [ -z "${{ steps.semver.outputs.version_tag }}" ]; then
+            echo "No conventional commits found - skipping release"
+            echo "skip=true" >> $GITHUB_OUTPUT
+          else
+            echo "Version to release: ${{ steps.semver.outputs.version_tag }}"
+            echo "skip=false" >> $GITHUB_OUTPUT
+          fi
+          
+      - name: Set up Go
+        if: steps.check.outputs.skip == 'false'
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)